antenna-documentation/src/site/markdown/generators/sw360-update-generator-step.md

```diff
@@ -56,3 +56,8 @@ the antenna configuration file will be used.
 
 #### Data Model
 You can find a description of the data model mapping in the [SW360 data model](../sw360-data-model.html) section.
+
+**Note**:
+In the current implementation of SW360, if the release to be updated contains a valid `Download URL`, 
+its `Component Type` is of `OSS` and the SW360 user has sufficient writing rights, the ECC (Export Control & Customs)
```

Comment:
A note might be worth it here indicating that antenna sets everything that is not specifically proprietary to "OSS" in SW360 Components (see: SW360ComponentAdapterUtils line 30)

Reply:
what are sufficient writing rights? does the everyday `USER` have them, or only `CLEARING ADMIN`? ([userGroup](https://github.com/eclipse/sw360/blob/master/libraries/lib-datahandler/src/main/thrift/users.thrift#L17-L24) can be found in users.thrift  in SW360)